org.clawsmail.Claws-Mail.json

```diff
@@ -0,0 +1,161 @@
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Yeraze/ytnef.git",
+          "commit": "d9b2f01d125c9b178f7a00ec397cc2a9f258e8e1"
+        }
+      ]
+    },
+    {
+      "name": "libopenjpeg",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/uclouvain/openjpeg.git",
+          "commit": "57096325457f96d8cd07bd3af04fe81d7a2ba788"
+        }
+      ]
+    },
+    {
+      "name": "libpoppler-glib",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://anongit.freedesktop.org/git/poppler/poppler.git",
+          "commit": "d2a37632661d1c8c29fa28a6e1a5d812508f4ea3"
+        }
+      ]
+    },
+    {
+      "name": "libical",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libical/libical.git",
+          "commit": "1b3eb45a7db4a9fd0aaf70fc7f8cfa9a4800ece4"
+        }
+      ]
+    },
+    {
+      "name": "libgumbo",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/google/gumbo-parser.git",
+          "commit": "3973c58d759574f2899528d2b3379e17d66dbcad"
+        }
+      ]
+    },
+    {
+      "name": "liboauth",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/x42/liboauth.git",
+          "commit": "186fb14843c1bc98b70ece02c74b5696b3de9ce7"
+        }
+      ],
+      "build-commands": [
+        "autoreconf --install",
+        "./configure --prefix=/app",
+        "make",
+        "make install"
+      ]
+    },
+    {
+      "name": "libuhttpmock",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.com/uhttpmock/uhttpmock.git",
+          "commit": "cda4b6330c6543c48600dd015af44bd0c2a21b4d"
+        }
+      ]
+    },
+    {
+      "name": "libgdata",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dgoa=disabled"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/GNOME/libgdata.git",
+          "commit": "f54d68f961015f85595b956ff0143a6ee0c41ff7"
+        }
+      ]
+    },
+    {
+      "name": "bogofilter",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd ./bogofilter && echo \"# Skip for Flatpak build\" > doc/Makefile.am && autoreconf -fi && ./configure --prefix=/app --disable-rpath --with-database=sqlite3 && make && make install"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://git.code.sf.net/p/bogofilter/git",
+          "branch": "75ef6fc25d7055321e3e7c3719732eb2fbcb9c6a"
+        }
+      ]
+    },
+    {
+      "name": "claws-mail",
+      "buildsystem": "autotools",
```

Comment:
Autotools is already the default buildsystem.

Reply:
Do you prefer to remove it and go with the implicit default?

Reply:
Up to you